docs/pipelineruns.md

```diff
@@ -82,6 +82,18 @@ spec:
 
 ```
 
+You may also use a `Tekton Bundle` to reference a pipeline defined remotely.
+
+ ```yaml
+ spec:
+   pipelineRef:
+     name: mypipeline
+     bundle: docker.com/myrepo/mycatalog:v1.0
```

Comment:
It sounded like you addressed Dan's comments about index.docker.io and digests, but I don't see the changes reflected here. Do you need to push those?

Reply:
Ah forgot this one. My bad.